R/translations.R

```diff
@@ -175,8 +180,10 @@ translations_direct_tidyverse <- list(
   char_length = str2lang("stringr::str_length"),
   character_length = str2lang("stringr::str_length"),
   concat = str2lang("stringr::str_c"),
+  len = str2lang("stringr::str_length"),
```

Comment:
Same comment as above, but in the tidyverse case the translation would be `stringr::str_length(stringr::str_trim(x, side = "right"))`